### ISSUE TITLE
Getters cannot work as before that in case of multi level nested structs with optional fileds.

### ISSUE THREAD

**Issue by verichenn** [tact-lang/tact#690]
#### Getter cannot be parsed correctly with error `Not a tuple`.
```
import "@stdlib/deploy";

struct Point {
    x: Int as int64;
    y: Int as int64;
}
 
struct Line {
    start: Point;
    end: Point;
}

struct Location {
     idx: Int;
     line: Line?;
}

contract Test with Deployable {

    get fun location(): Location {
        let start: Point = Point{ x: 0, y: 0};
        let end: Point = Point{ x: 1, y: 1};

        return Location {
            idx: 1,
            line: Line {start: start, end: end}
        }
    }
}
```

```typescript
import { Blockchain } from "@ton/sandbox";
import { Test } from "../build/Test/tact_Test";
import { toNano } from "@ton/core";

describe('TEST', () => {

    it('1', async () => {
        let blockchain = await Blockchain.create({});

        let deployer = await blockchain.treasury('deployer');
        let test = blockchain.openContract(await Test.fromInit());
        
        await test.send(
            deployer.getSender(),
            {
                value: toNano('0.1'),
            },
            {
                $$type: 'Deploy',
                queryId: 0n,
            }
        );
        await test.getLocation(); //Not a tuple
    });
});
```

**Comment by anton-trunov** [comment]
Can you provide more info please? Are you experiencing this in TypeScript tests? What is your test code then?

**Comment by verichenn** [comment]
If the field `line` of `Location` is non optional, the getter will run correctly.

**Comment by verichenn** [comment]
and it works in v1.4.0

**Comment by Gusarich** [comment]
I see the problem, turns out that the 1.4.2 fix is incomplete.

**Comment by anton-trunov** [comment]
@verichenn Thanks a lot for all your bug reports! Really valuable. Btw, we just released Tact v1.4.3 on NPM. Would be great if you could try the new version and share your feedback

**Comment by verichenn** [comment]
> @verichenn Thanks a lot for all your bug reports! Really valuable. Btw, we just released Tact v1.4.3 on NPM. Would be great if you could try the new version and share your feedback

Thanks for the update! Everything seems fine now.

**Comment by anton-trunov** [comment]
Awesome! Thanks for trying it out, really appreciate it
